### PR TITLE
Load Balancing with Fabio Guide

### PR DESCRIPTION
**Pull request by Omar-Khawaja**
Added load-balancing guide and edited Guides drop down menu to reflect that.

### CODE REVIEW

File: website/source/guides/load-balancing/load-balancing.html.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  There are multiple approaches to load balancing within a Nomad cluster. This
+  discusses the most popular strategies.
+---
+
+# Load Balancing
+
+There are multiple approaches to set up load balancing across a Nomad cluster.
+Most of these methods assume Consul is installed alongside Nomad.
+
+- [Load Balancing Strategies for Consul](https://www.hashicorp.com/blog/load-balancing-strategies-for-consul)
```
Comment (rcgenova) [inline website/source/guides/load-balancing/load-balancing.html.md]: Can we link to the Consul LB strategies inline and instead use a bullet to link directly to the Fabio guide?

Comment (shantanugadgil) [reply website/source/guides/load-balancing/load-balancing.html.md]: How about adding links to Gobetween as well:
http://gobetween.io/

http://gobetween.io/documentation.html#consul

File: website/source/guides/load-balancing/fabio.html.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+[apache]: https://httpd.apache.org/
+[classic-lb]: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html
+[classic-lb-doc]: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-getting-started.html
+[elb]: https://aws.amazon.com/elasticloadbalancing/
+[fabio]: https://fabiolb.net/
+[lb-homepage]: /assets/images/lb-homepage.png
+[lb-routing-table]: /assets/images/lb-routing-table.png
+[routing-table]: /assets/images/routing-table.png
+[system]:https://www.nomadproject.io/docs/runtime/schedulers.html#system
```
Comment (rcgenova) [inline website/source/guides/load-balancing/fabio.html.md]: Broken link .. should be `/docs/schedulers.html#system`. Also, you can use a relative link for this.

File: website/source/layouts/guides.erb [hunk 3]
```diff
@@ -65,6 +65,15 @@
         <a href="/guides/namespaces.html">Namespaces</a>
       </li>
 
+      <li<%= sidebar_current("guides-load-balancing") %>>
```
Comment (rcgenova) [inline website/source/layouts/guides.erb]: Lets place the `Load Balancing` guides section navigation link right above `Web UI`. We moved away alphabetical sorting with the new reorganization.

File: website/source/guides/load-balancing/fabio.html.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+job "fabio" {
+  datacenters = ["dc1"]
+  type = "system"
+
+  group "fabio" {
+    task "fabio" {
+      driver = "docker"
+      config {
+        image = "fabiolb/fabio"
+        network_mode = "host"
```
Comment (analytically) [inline website/source/guides/load-balancing/fabio.html.md]: this is the default

**Pull request merged**